script.js

```diff
@@ -1,11 +1,18 @@
 // Create an array called `colors` that contains three colors.
-
+var colors = ['red', 'yellow', 'green']
 // Log the last item in the array to the console.
-
+console.log(colors[2])
 // Create a `for` loop that prints out "My favorite color is [insert color here]" for each item in `colors`.
-
+for (i=0;i<colors.length; i++){
+  console.log("My favorite color is " + colors[i])
+}
 // Create an `instructor` object that contains three key-value pairs.
-
+var instructor = {
+  age: 25,
+  interest: "coding",
+  build: "tall and lanky",
+}
 // Add a `facial-hair` property to `instructor`.
-
+instructor.facialHair = "scruff"
```

Comment:
Looking for you to use a hyphen in the key name. So adding that key-value pair would look something like...

```js
instructor["facial=hair"] = "scruff";
```